content/applications/services/helpdesk/overview/reports.rst

```diff
@@ -1,60 +1,188 @@
+- Hours Open
+- Hours Spent
+- Hours to Assign
+- Hours to Close
+- Hours to First Response
+- Hours to SLA Deadline
+- Rating /5
+- Count
+
+SLA Status Analysis
+-------------------
+
+The :guilabel:`SLA Status Analysis` report tracks how quickly an :abbr:`SLA (Service Level
+Agreement` is fulfilled, as well as the success rate of individual policies. By default, the report
+is filtered to show the number of :abbr:`SLAs (Service Level Agreements)` failed, as well as the
+failure rate over the last 30 days, grouped by team.
+
+.. image:: reports/reports-sla-status.png
    :align: center
-   :alt: View of a helpdesk ticket analysis by team and ticket type in Odoo Helpdesk
+   :alt: View of Group by options of Ticket Analysis report
+
+To change the measures used in the report, click :guilabel:`Measures` and choose from the drop-down
+menu. The available measures for the :guilabel:`SLA Status Analysis` report include:
+
+- % of Failed SLA
+- % of SLA in Progress
+- % of Successful SLA
+- Number of SLA Failed
+- Number of SLA Successful
+- Number of SLA in Progress
+- Working Hours to Assign
+- Working Hours to Close
+- Working to Reach SLA
+- Count
+
+.. example::
+   To see the number of tickets that were able to achieve the stated :abbr:`SLA (Service Level
+   Agreement)` objectives, and track the amount of time it took to achieve those objectives, click
+   :menuselection:`Measures --> Number of SLA Successful` and :menuselection:`Measures --> Workings
+   Hours to Reach SLA`. To sort these results by the team members assigned to the tickets, select
+   :menuselection:`Total --> Assigned to`.
+
+.. seealso::
+   - :doc:`sla`
+
+Customer Ratings
+----------------
 
-- The number of tickets closed per day, per team.
+The :guilabel:`Customer Ratings` report displays an overview of the ratings received on individual
+support tickets, as well as any additional comments submitted with the rating.
+
+.. image:: reports/reports-customer-ratings.png
+   :align: center
+   :alt: View of the kanban display in the Customer Ratings report
 
-Get an overview of how many requests each team is closing per day in order to measure their
-performance. Identify productivity levels to understand how many requests they are able to handle.
+Click on an individual rating to see additional details about the rating assigned by the customer,
+including a link to the original ticket.
 
-.. image:: reports/report_close_date_team.png
+.. image:: reports/reports-ratings-details.png
    :align: center
-   :alt: View of a helpdesk ticket analysis by team and close date in Odoo Helpdesk
+   :alt: View of the details of an individual customer rating
 
 .. tip::
-   Filter it by *Assignee* to see Key Performance Indicators (KPI) per agent.
+   Select :guilabel:`Visible Internally Only` to hide a rating from the customer portal.
+
+The :guilabel:`Customer Ratings` report is displayed in a kanban view by default, but can also be
+displayed in graph or pivot view.
```

Comment:
```suggestion
The :guilabel:`Customer Ratings` report page is displayed in a kanban view by default, but can also be
displayed in graph, list, or pivot view.
```

Unsure if those view options should have a :guilabel:, since they are, technically, visible/clickable options on the page, itself. Might be worth looking into.

Reply:
confirmed with @samueljlieber they do not need a guilabel 👍